src/pages/home/report/ReportActionCompose.js

```diff
@@ -379,10 +384,10 @@ class ReportActionCompose extends React.Component {
      * @param {Boolean} shouldDebounceSaveComment
      */
     updateComment(newComment, shouldDebounceSaveComment) {
-        this.setState({
-            isCommentEmpty: !!newComment.match(/^(\s|`)*$/),
-            value: newComment,
-        });
+        const isCommentEmpty = !!newComment.match(CONST.REGEX.IS_COMMENT_EMPTY);
```

Comment:
This style guideline we maybe haven't standardized, but I always prefer this more explicit approach:

```suggestion
        const isCommentEmpty = Boolean(newComment.match(CONST.REGEX.IS_COMMENT_EMPTY));
```